pages/contribute.js

```diff
@@ -99,6 +99,8 @@ class TiersPage extends React.Component {
     const waysToContribute = [];
     const canContribute = collective.isActive && collective.host;
     const hasContributors = this.hasContributors(collective.contributors, collective.events);
+    const hasEventContributors = this.hasContributors([], collective.events);
+    const hasProjectContributors = this.hasContributors([], collective.projects);
```

Comment:
I don't know how to create a connected collective locally to test my changes, the [docs mention ](https://docs.opencollective.com/help/collectives/connectedcollectives)that it is a manual process. Is there any collective with child collectives on stage env already that I can use?

Reply:
You can try this one: https://staging.opencollective.com/open-food-network/connected-collectives.

Locally, it's either done by Root Actions > Connected Collectives or by inserting the `Members` entry directly.